learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+```
+layout: getting-started-with-ballerina-left-nav-pages
+title: Getting Started with Ballerina
+description: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+keywords: ballerina, programming language, ballerina packages, getting started
+permalink: /learn/getting-started-with-ballerina
+active: getting-started-with-ballerina
+intro: Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+---
+```
+
+Let’s get you set up a Ballerina development environment and write a simple Ballerina program.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- A command terminalA text editor.
+- [VSCode](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed is our preferred choice, and it has good support for Ballerina.
```

Comment:
Since we have mentioned about text editor before shouldn't we list these as one is optional. Basically a text editor or VSCode Ballerina extension? We don't need both right?

Reply:
Fixed. Thanks